src/Components/Footer/Footerlink/index.css

```diff
@@ -0,0 +1,8 @@
+.Footerlink{
+    font-size: 16px;
+    line-height: 22px;
+    font-family: Euclid Flex;
```

Comment:
In `font-family: Euclid Flex;` you should use quotes around the typeface name. Also, maybe have a fallback such as `font-family: 'Euclid Flex', sans-serif;`?